sdks/python/apache_beam/io/fileio_test.py

```diff
@@ -348,57 +346,6 @@ def test_write_entire_file(self):
       self.assertEqual(f.read().splitlines(), lines)
 
 
-class TestTextFileSink(unittest.TestCase):
```

Comment:
Though TextFileSink is deprecated, do we want to remove these tests before the TextFileSink implementation is removed?